src/main/java/com/synopsys/integration/alert/web/api/certificate/CertificateActions.java

```diff
@@ -81,7 +81,7 @@ public CertificateActions(CertificatesDescriptorKey descriptorKey, Authorization
     }
 
     @Override
-    public ActionResponse<CertificateModel> readWithoutChecks(Long id) {
+    protected ActionResponse<CertificateModel> readWithoutChecks(Long id) {
```

Comment:
Are we changing these methods to protected so that only the controller in the same package can call these methods?

Reply:
All of the abstract methods in AbstractResourceActions are protected. While working on users we noticed this was a mistake and after checking with Paulo I decided to include the changes in this PR as well.

Reply:
It was a mistake this wasn't changed initially.